core/src/main/scala/org/apache/spark/ui/exec/ExecutorsTab.scala

```diff
@@ -38,47 +37,71 @@ private[ui] class ExecutorsTab(parent: SparkUI) extends SparkUITab(parent, "exec
   }
 }
 
+case class ExecutorTaskSummary(
```

Comment:
One more thing occurred to me: do we need to make this `private[ui]`? I think so.